automation_oca/tests/test_automation_activity.py

```diff
@@ -1,12 +1,27 @@
 # Copyright 2024 Dixmit
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-
+from odoo import Command
 from odoo.tests import Form
 
 from .common import AutomationTestCase
 
 
 class TestAutomationActivity(AutomationTestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = cls.env["res.users"].create(
```

Comment:
You don't need to create manually a user, use:

```python
from odoo.tests import new_test_user

cls.user = new_test_user(cls.env, login="...", groups="...")
```

Reply:
Fixed